<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pB_hinzufuegen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="statusTip">
         <string>Neues Item hinzufügen</string>
        </property>
        <property name="text">
         <string>Hinzufügen</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icon/emblem-new.svg</normaloff>:/icon/emblem-new.svg</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::Batched</enum>
        </property>
        <property name="gridSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
        <item>
         <property name="text">
          <string>Äpfel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Birnen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Doesenwurst</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icon/emblem-synchronizing.svg</normaloff>:/icon/emblem-synchronizing.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="ItemDetailframe">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="ItemDetailLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="detailIcon">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="icons.qrc">:/icon/spoon-and-a-fork.svg</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_DetailName_2">
                 <property name="font">
                  <font>
                   <pointsize>24</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                   <stylestrategy>PreferAntialias</stylestrategy>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="LagerplatzLayout">
               <item>
                <widget class="QLabel" name="lbl_DetailMainLagerplatz">
                 <property name="text">
                  <string>MainLagerplatz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_DetailPfeilLocation">
                 <property name="text">
                  <string>&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_DetailSubLagerplatz">
                 <property name="text">
                  <string>Sublagerplatz</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="lbl_DetailMHD_2">
                 <property name="text">
                  <string>MHD:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_DetailMHD">
                 <property name="text">
                  <string>MHD-Datum</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="lbl_DetailMenge">
                   <property name="text">
                    <string>Menge:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="le_DetailMenge">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="le_DetailPortionen">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Portionen:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Anzahl:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="spinBox">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="wrapping">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>kcal/Portion:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="le_kcalPortion">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tW_Items">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuProdukte">
    <property name="title">
     <string>Produkte</string>
    </property>
    <widget class="QMenu" name="menuNahrungsmittel">
     <property name="title">
      <string>Nahrungsmittel</string>
     </property>
     <widget class="QMenu" name="menuKategorien_2">
      <property name="title">
       <string>Kategorien</string>
      </property>
      <addaction name="actionSonstiges_2"/>
     </widget>
     <addaction name="actionAddFood"/>
     <addaction name="menuKategorien_2"/>
     <addaction name="actionAlles_anzeigen_2"/>
    </widget>
    <addaction name="menuNahrungsmittel"/>
    <addaction name="actionNon_Food"/>
    <addaction name="actionHinzuf_gen_2"/>
   </widget>
   <widget class="QMenu" name="menuLagerorte">
    <property name="title">
     <string>Lagerorte</string>
    </property>
    <addaction name="actionKategorienBearbeiten"/>
   </widget>
   <widget class="QMenu" name="menuKategorien">
    <property name="title">
     <string>Kategorien</string>
    </property>
    <addaction name="actionSonstiges"/>
    <addaction name="separator"/>
    <addaction name="actionBearbeiten"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuProdukte"/>
   <addaction name="menuLagerorte"/>
   <addaction name="menuKategorien"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionAddFood"/>
  </widget>
  <action name="actionNon_Food">
   <property name="text">
    <string>Non-Food</string>
   </property>
  </action>
  <action name="actionSonstiges">
   <property name="text">
    <string>Sonstiges</string>
   </property>
  </action>
  <action name="actionBearbeiten">
   <property name="text">
    <string>Bearbeiten</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="statusTip">
    <string>Beendet das Programm</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAddFood">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/emblem-new.svg</normaloff>:/icon/emblem-new.svg</iconset>
   </property>
   <property name="text">
    <string>Hinzufügen...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAlles_anzeigen_2">
   <property name="text">
    <string>Alles anzeigen...</string>
   </property>
  </action>
  <action name="actionSonstiges_2">
   <property name="text">
    <string>Sonstiges</string>
   </property>
  </action>
  <action name="actionHinzuf_gen_2">
   <property name="icon">
    <iconset>
     <normaloff>../icons/emblem-new.svg</normaloff>../icons/emblem-new.svg</iconset>
   </property>
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/emblem-synchronizing.svg</normaloff>:/icon/emblem-synchronizing.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Daten neu einlesen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionKategorienBearbeiten">
   <property name="text">
    <string>Bearbeiten...</string>
   </property>
   <property name="toolTip">
    <string>Kategorien bearbeiten</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
